org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -36,17 +36,29 @@ class StateReader {
     }
 }
 
+// Helper method to convert Wink's 0.0-1.0 brightness scale to a 0-100 scale
+function scaleBrightnessToDim(brightnessValue) {
+    return Math.floor(brightnessValue * 100);
+}
+
+// Helper method to convert a 0-100 scale to Wink's 0.0-1.0 brightness scale
+function scaleDimToBrightness(dimmingValue) {
+    return dimmingValue / 100;
```

Comment:
That was my first thought as well, but I tried it out and it's fine as is. (e.g. 43 / 100 = 0.43; C-style integer/floating point rules don't apply in JavaScript, I suppose)

---
In reply to: [76938955](https://github.com/openT2T/translators/pull/87#discussion_r76938955) [](ancestors = 76938955)